src/modules/Popup/Popup.js

```diff
@@ -1,7 +1,8 @@
 import React, { Component, PropTypes } from 'react'
 import { findDOMNode } from 'react-dom'
 import { META } from '../../lib'
-import Portal from 'react-portal'
+import Portal from '../../addons/Portal/Portal'
+import classnames from 'classnames'
```

Comment:
We're using `cx` as the import name for `classnames`.  Let's update for consistency.